spring-batch-infrastructure/src/main/java/org/springframework/batch/item/adapter/AbstractMethodInvokingDelegator.java

```diff
@@ -45,7 +45,7 @@
 
 	private String targetMethod;
 
-	private Object[] arguments;
+	protected Object[] arguments;
```

Comment:
Why would we make this field protected instead of adding a getter?

Reply:
Using a getter is better indeed. PR updated. Thanks for the review!